src/getDeviceInfo.js

```diff
@@ -1,19 +1,19 @@
 import React from "react";
-import DeviceInfo from 'react-native-device-info';
+import DeviceInfo from "react-native-device-info";
 
-function GetDeviceInfo(props) {
-  let deviceJSON = {}
-  deviceJSON.uniqueId = DeviceInfo.getUniqueId();
-  deviceJSON.deviceId = DeviceInfo.getDeviceId();
-  deviceJSON.deviceType = DeviceInfo.getDeviceType();
-  deviceJSON.model = DeviceInfo.getModel();
-  deviceJSON.brand = DeviceInfo.getBrand();
-  deviceJSON.systemName = DeviceInfo.getSystemName();
-  deviceJSON.systemVersion = DeviceInfo.getSystemVersion();
-  deviceJSON.isTablet = DeviceInfo.isTablet();
-  deviceJSON.appName = DeviceInfo.getApplicationName();
+const getDeviceInfo = () => {
+  const device = {};
+  device.uniqueId = DeviceInfo.getUniqueId();
+  device.deviceId = DeviceInfo.getDeviceId();
+  device.deviceType = DeviceInfo.getDeviceType();
+  device.model = DeviceInfo.getModel();
+  device.brand = DeviceInfo.getBrand();
+  device.systemName = DeviceInfo.getSystemName();
+  device.systemVersion = DeviceInfo.getSystemVersion();
+  device.isTablet = DeviceInfo.isTablet();
+  device.appName = DeviceInfo.getApplicationName();
 
-  return JSON.stringify(deviceJSON);
+  return JSON.stringify(device);
```

Comment:
I think that you should return here the `device` object, and you can stringify it whenever you need that. This change will help you with the callback feature, where you should send an object, not a JSON.